### ISSUE TITLE
Run-Time Error 53: File Not Found: libEdge.dll

### ISSUE THREAD

**Issue by KallunWillock** [peakpeak-github/libEdge#3]
Hello. 
Thank you very much for trying to extend Webview2 to the VBA community. I may well be that I'm doing something fundamentally wrong, but I can't get it to work on my system. I'm using Office 365 - Excel 64-bit.  It keeps breaking on the following line:

`DoDemo = VBA_StartEdge()             ` 
with a runtime error 53 (libEdge.dll not found).

I've tried using the full path to the DLL, I've tried changing directories - is there something I'm missing? Thank you!

**Comment by peakpeak-github** [comment]
Sorry, should have mentioned that, if using 64-bit Office you need to copy libEdge/x64/Release/libEdge.dll and libEdge/x64/Release/WebView2Loader.dll to libEdge/VBA_libEdge. The current DLLs are 32-bit.

**Comment by KallunWillock** [comment]
Hi, thank you for this. I only just came back to this now and saw your comment. It works, thank you. The window closes immediately after loading, which I assume was the intention, so I may have to play around with it to figure it out some. But thank you kindly!

**Comment by peakpeak-github** [comment]
The window should stay open until you click on Ok in the messagebox.
![image](https://user-images.githubusercontent.com/10742803/121299790-3b758680-c8f6-11eb-8a01-cb63250d5a89.png)

**Comment by KallunWillock** [comment]
Thank you very much for responding. 
So I went to test it on another computer - same result. I eventually worked out that on loading (on my computers, at least), the window immediately hides behind the window beneath it.  I suspect that's why you set the visibility of the Excel Application to False at the outset - my problem was that I had a second instance of Excel running, so the Window would constantly disappear behind it. 
Thank you again.
